package.json

```diff
@@ -980,9 +980,14 @@
             },
             "diagnostics": {
               "type": "boolean",
-              "default": false,
+              "default": true,
               "description": "If true, the language server will provide build, vet errors and the extension will ignore the `buildOnSave`, `vetOnSave` settings."
             },
+            "documentLink": {
+              "type": "boolean",
+              "default": true,
+              "description": "If true, the language server's document link feature will be enabled."
```

Comment:
Can you update the description to include a little more details on what this feature would mean to the end user?

Reply:
Done.